scripts/code-web.js

```diff
@@ -20,17 +20,35 @@ const EXTENSIONS_ROOT = path.join(APP_ROOT, 'extensions');
 const WEB_MAIN = path.join(APP_ROOT, 'src', 'vs', 'code', 'browser', 'workbench', 'workbench-dev.html');
 
 const args = minimist(process.argv, {
-	string: [
+	boolean:[
 		'no-launch',
+		'help'
+	],
+	string: [
 		'scheme',
-		'host'
+		'host',
+		'port',
+		'local_port'
 	],
-	number: [
-		'port'
-	]
 });
 
+if(args.help){
+	console.log(
+		'yarn web [options]\n' +
+			' --no-launch   Launch browser\n' +
```

Comment:
I think the text is wrong, if specified this does not launch the browser, right?